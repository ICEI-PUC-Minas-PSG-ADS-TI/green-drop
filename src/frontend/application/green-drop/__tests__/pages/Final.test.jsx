import React from 'react';
import { render } from '@testing-library/react-native';
import renderer from 'react-test-renderer';
import Final from '../../pages/contribuir/final/index.jsx';

describe('<Final />', () => {
it('renderiza corretamente', () => {
render(<Final />);
});

it('snapshot de Final', () => {
const tree = renderer.create(<Final />).toJSON();
expect(tree).toMatchSnapshot();
});
});
