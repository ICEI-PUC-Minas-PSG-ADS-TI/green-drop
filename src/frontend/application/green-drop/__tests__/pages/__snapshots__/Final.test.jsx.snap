// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Final /> snapshot de Final 1`] = `null`;
